### PR TITLE
Helper ontology extraction

### PR DESCRIPTION
**Pull request by cmader**
extract helper ontologies for API generation and validation from core information model.
(addresses https://github.com/IndustrialDataSpace/InformationModel/issues/2)

### CODE REVIEW

**Comment by seebi** [comment]
the name "The Industrial Dataspace Information Model Validation Ontology" is imho strongly misleading with regard to what it is - can we rename this?

File: util/validation.ttl [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+@prefix valid: <http://industrialdataspace.org/2017/01/ids/validation#> .
+
+# Description of this ontology
+# ----------------------------
+
+valid: a owl:Ontology;
+    dct:title "The Industrial Dataspace Information Model Validation Ontology"@en;
+    rdfs:comment "An ontology to annotate RDF predicates with information needed to generate an API for the Information Model.";
```
Comment (seebi) [inline util/validation.ttl]: the name "The Industrial Dataspace Information Model Validation Ontology" is imho strongly misleading with regard to what it is - can we rename this?

**Comment by seebi** [comment]
btw: according to the contribution guide, we use prefixed branch names (feature/bugfix/release)

**Comment by cmader** [comment]
sorry about violating the naming conventions, will stick to them the next time!

File: util/validation.ttl (hunk 1 above)
Comment (cmader) [reply util/validation.ttl]: As these properties are currently relevant for transforming the ontology into a java-based representation (suitable for developers to easily create instances), we could rename it to "IDS Transformation Hints Ontology". Would that be more clear?

**Comment by cmader** [comment]
created a new repository that contains the helper ontologies: https://github.com/IndustrialDataSpace/InformationModelHelperOntologies

**Pull request closed without merging**